recipes/snakemake-minimal/meta.yaml

```diff
@@ -28,7 +28,7 @@ requirements:
     - appdirs
     - datrie
   run:
```

Comment:
Since this is the "minimal" package I think it should only include the packages also listed in `Setup.py`. So maybe drop `psutil`

Reply:
Good catch. However, psutil will soon become a requirement (a future version of Snakemake will do benchmarking by default). Hence, I think it should be fine to keep it.